.dumi/theme/slots/Header/index.tsx

```diff
@@ -41,13 +41,15 @@ const useStyle = createStyles(({ token, css }) => {
   const searchIconColor = '#ced4d9';
   return {
     header: css`
-      position: sticky;
+      position: fixed;
       top: 0;
       z-index: 1000;
       max-width: 100%;
       background: ${token.colorBgContainer};
       box-shadow: ${token.boxShadowTertiary};
       backdrop-filter: blur(8px);
+      width: 100dvw;
```

Comment:
```suggestion
      width: 100%;
```

用了个这么新的动态视口单位吗， 100% 会有问题吗